app/assets/javascripts/app/pages/admin_pods.js

```diff
@@ -16,11 +54,17 @@ app.pages.AdminPods = app.views.Base.extend({
 
     // avoid reflowing the page for every entry
     var fragment = document.createDocumentFragment();
+    this.$("tbody").empty();
+
     this.pods.each(function(pod) {
-      self.rows.push(new app.views.PodEntry({
-        parent: fragment,
-        model: pod
-      }).render());
+      if (self.podfilter === "" ||
+        self.podfilter === "active" && pod.get("status") === "no_errors" ||
```

Comment:
Maybe change this one to check the `active` field instead? This would then include all pods which are being handled as "active" in the backend (all pods to which is being federated). This also includes pods with warnings (but which are still online and working), and pods which only have errors since less than 14 days (so maybe only a temporary downtime?).

If you want to filter for "no error" then the label on the button text should probably say something like "no error", because "active" means something different.